### PR TITLE
Replace Gem eco with ejs

### PR DESCRIPTION
**Pull request by dinesh-skyach**
[WIP] #162 Removed gem eco

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -95,3 +94,4 @@ group :test do
   gem 'rack_session_access', '~> 0.1'
   gem 'webmock',             '~> 3.3'
 end
+gem 'ejs'
```
Comment (yivo) [inline Gemfile]: @dinesh-skyach please don't put gem at the end. Put in better place. And lock version range.

File: app/assets/javascripts/templates/market_trade.jst.ejs [hunk 2]
```diff
@@ -1,16 +1,16 @@
-<% trend = formatter.trend @type %>
+<% trend = formatter.trend(type) %>
 
-<tr id='market-trade-<%= @tid %>' class='<%= @classes %>'>
+<tr id='market-trade-<%= tid %>' class='<%= classes %>'>
```
Comment (yivo) [inline app/assets/javascripts/templates/market_trade.jst.ejs]: @dinesh-skyach Could you please use double quotes for HTML attributes? This is out style convention.

**Review by yivo — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (dinesh-skyach) [reply Gemfile]: @yivo done

File: app/assets/javascripts/templates/market_trade.jst.ejs (hunk 2 above)
Comment (dinesh-skyach) [reply app/assets/javascripts/templates/market_trade.jst.ejs]: @yivo done

File: Gemfile [hunk 3]
```diff
@@ -70,6 +69,7 @@ gem 'memoist', '~> 0.16'
 gem 'method-not-implemented', '~> 1.0'
 gem 'passgen', '~> 1.0'
 gem 'validates_lengths_from_database', '~> 0.7.0'
+gem 'ejs','~> 1.1.1'
```
Comment (yivo) [inline Gemfile]: @dinesh-skyach I think `~> 1.1` will be enough.

**Review by yivo — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 3 above)
Comment (dinesh-skyach) [reply Gemfile]: @yivo, I have changed the version to 1.1

**Review by yivo — APPROVED** [review]

**Pull request merged**